src/presentation/styles/colors.pcss

```diff
@@ -32,6 +32,98 @@
    */
   --color-line: var(--color-gray-superlight);
   --color-delimiter-light: var(--color-transparent);
+
+  /**
+   * Classic Dark Colors
+   */
+   --bg-primary-classic: #242527;
```

Comment:
Maybe add `color` prefix/postfix to color variables to make their purpose more clear?

Reply:
I don't think so, its better to stay with the same naming as Figma